serial_linux.go

```diff
@@ -60,11 +60,45 @@ func openPort(name string, baud int, readTimeout time.Duration) (p *Port, err er
 		}
 	}()
 
+	// Base settings
+	cflagToUse := syscall.CREAD | syscall.CLOCAL | rate
+	switch databits {
+	case byte(DATABITS_8):
+		cflagToUse |= syscall.CS8
+	case byte(DATABITS_7):
+		cflagToUse |= syscall.CS7
+	case byte(DATABITS_6):
+		cflagToUse |= syscall.CS6
+	case byte(DATABITS_5):
+		cflagToUse |= syscall.CS5
+	default:
+		cflagToUse |= syscall.CS8
+	}
+	// Stop bits settings
+	switch stopbits {
+	case byte(STOPBITS_2):
+		cflagToUse |= syscall.CSTOPB
+	case byte(STOPBITS_15):
+		// Don't know how to set 1.5, use 2!
+		cflagToUse |= syscall.CSTOPB
```

Comment:
I think returning an error is better than silently doing something different than what was asked for.